.storybook/preview-head.html

```diff
@@ -209,4 +209,25 @@
 	.sbdocs footer {
 		text-align: center;
 	}
+
+	.aa-ItemLink,
+	.aa-ItemLink:hover,
+	.aa-ItemLink:focus,
+	.aa-ItemLink:active {
+		text-decoration: none;
+	}
+
+	.aa-ItemContentTitle {
+		font-weight: bold;
+	}
+
+	.aa-ItemContentCategory {
+		color: #ccc;
+	}
+
+	.aa-ItemContentDescription {
+		overflow: hidden;
+	}
```

Comment:
You seem to be overriding stuff from Algolia here but your link import comes afterwards?

Reply:
Fixed